h2o-core/src/main/java/water/rapids/ast/params/AstNumList.java

```diff
@@ -241,7 +241,8 @@ public boolean has(long v) {
    */
   public long index(long v) {
     int bIdx = findBase(v);
-    if (bIdx >= 0) return water.util.ArrayUtils.sum(_cnts, 0, bIdx - 1);
+//    if (bIdx >= 0) return water.util.ArrayUtils.sum(_cnts, 0, bIdx - 1);
```

Comment:
please remove the commented out line

Add a junit test for this method (to test the edge cases)